spec/factories/books.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :book do
+    title "MyString"
```

Comment:
OK! Please @cquiro delete the ApplicationRecord class and make the models inherit directly from ActiveRecord::Base, there's no point in having an extra class that does nothing just because it is generated by rails